examples/pytorch/dimenet/README.md

```diff
@@ -0,0 +1,125 @@
+num_blocks        int    Number of building blocks to be stacked.                               Default is 6 in DimeNet and 4 in DimeNet++   
+num_spherical     int    Number of spherical harmonics.                                         Default is 7   
+num_radial        int    Number of radial basis functions.                                      Default is 6   
+envelope_exponent int    Shape of the smooth cutoff.                                            Default is 5   
+cutoff            float  Cutoff distance for interatomic interactions.                          Default is 5.0 
+num_before_skip   int    Number of residual layers in interaction block before skip connection. Default is 1   
+num_after_skip    int    Number of residual layers in interaction block after skip connection.  Default is 2   
+num_dense_output  int    Number of dense layers for the output blocks.                          Default is 3   
+targets           list   List of targets to predict.                                            Default is ['mu']
+output_init       string Initial function name for output layer.                                Default is 'GlorotOrthogonal'
+```
+
+###### Training options
+```
+num_train         int   Number of training samples.                     Default is 110000
+num_valid         int   Number of validation samples.                   Default is 10000
+data_seed         int   Random seed.                                    Default is 42
+lr                float Learning rate.                                  Default is 0.001
+weight_decay      float Weight decay.                                   Default is 0.0001
+ema_decay         float EMA decay.                                      Default is 0.
+batch_size        int   Batch size.                                     Default is 100
+epochs            int   Training epochs.                                Default is 300
+early_stopping    int   Patient epochs to wait before early stopping.   Default is 20
+num_workers       int   Number of subprocesses to use for data loading. Default is 18
+gpu               int   GPU index.                                      Default is 0, using CUDA:0
+interval          int   Time intervals for model evaluation.            Default is 50
+step_size         int   Period of learning rate decay.                  Default is 100
+gamma             float Factor of learning rate decay.                  Default is 0.3
+```
+
+### Performance
+
+- Batch size is different
+- Linear learning rate warm-up is not used
+- Exponential learning rate decay is not used
+- Exponential moving average (EMA) is closed
+
+| Target | mu | alpha | homo | lumo | gap | r2 | zpve | U0 | U | H | G | Cv |
+| :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| MAE(DimeNet in Table 1)      | 0.0286 | 0.0469 | 27.8 | 19.7 | 34.8 | 0.331 | 1.29 | 8.02 | 7.89 | 8.11 | 8.98 | 0.0249 |
```

Comment:
Seems like the values for tasks like alpha should be x 10^-2?

Reply:
10 ^ -3 except Cv, mu, alpha, r2

Reply:
I guess you need to add a note for that.